boundaries/build/index.json

```diff
@@ -12,6 +12,18 @@
       "lang:en": "NAME_EN"
     }
   },
+  {
+    "directory": "flacs",
+    "area_type_wikidata_item_id": "Q1062177",
+    "associations": [],
```

Comment:
Should Q28172635 be an association in here?

Reply:
Ooh, yes, you're right :-)